src/RPCTestsFiller/AvailabilityTests/eth_getBalanceFiller.json

```diff
@@ -0,0 +1,13 @@
+{
+    "eth_getBalance" : {
+        "request_method" : "eth_getBalance",
+	"request_params" : [
+	     "0x1122334455667788991011121314151617181920",
+	     "0x01"
+	],
+	"expect_return" : {
+	     "//" : "exactly checks that returned json object is exactly the same as described here. including all subojects",
+	     "exactly" : "0x0"
```

Comment:
I'm thinking that json-schema would work well here rather than defining our own DSL.  Both of the `exactly` and `checkstring` can be expressed in json schema.

```javascript
{
  "type": "string",
  "enum": ["0x0"]
}
```

```javascript
{
   "type": "string"
}
```

If we want to leave this open to be extensible then we could define a `meta` field that defines the validation engine and start with just `jsonschema` as our only validation engine.

Reply:
this is just a notation. but right having an engine field would be even better.
so I could change it to have json shema in the filler itself.